docs/Getting Started/configuration.md

```diff
@@ -58,26 +50,26 @@ The configuration structure for __Codyze v3__ separates the options by subcomman
 ```
 The value of the option is taken from the object which corresponds to the subcommand used for the execution.
 This means if `codyze analyze` is called, source would be `src`, and if `codyze lsp` is called, source would be `other-src`.
-An exemplary configuration file for __Codyze v3__ can also be found in the [GitHub repository](https://github.com/Fraunhofer-AISEC/codyze/blob/main/codyze-v3/codyze/config.json).
+An exemplary configuration file can also be found in the [GitHub repository](https://github.com/Fraunhofer-AISEC/codyze/blob/main/codyze-cli/config.json).
 
 ## List of Configurations
-This is a list of all available configurations, their descriptions and their respective name in __Codyze v2__ and __Codyze v3__.
+This is a list of all available configurations, their descriptions and their respective name.
 The names are the same for the configuration file and the CLI options.
 
 `./` denotes the working directory in which Codyze was started.
 
-| Codyze v2               | Codyze v3               | Value               | Description                                                                                                                                                                                                                                               | Default Value     |
+| Legacy Codyze           | Redesigned Codyze       | Value               | Description                                                                                                                                                                                                                                               | Default Value     |
```

Comment:
We can remove the whole first column with legacy Codyze.

Reply:
Done in 836e36ac0f3a919b50b132a1e16f102b3dd951a1.